pcdscalc/misc_calcs.py

```diff
@@ -0,0 +1,351 @@
+import numpy as np
+from .be_lens_calcs import alias
+
+# TODO: find a better number
+_manual_energy = 9.8
+
+
+def set_energy(energy):
+    """
+    Set a global, manual, working energy for doing offline calcs.
+    """
+    global _manual_energy
+    _manual_energy = energy
+
+
+def check_id(material_id):
+    """Check to see if you are using an alias. Return the chemical formula."""
+    try:
+        return alias[material_id]
+    except Exception:
+        return material_id
+
+
+def cosd(angle):
```

Comment:
i'm wondering now if `numpy` has something for `cosd` and i don't know about it...

Reply:
Jumping back and forth from radians/degrees unnecessarily is not a good thing for floating point calculations, and the library functions all use radians internally as far as I know. It's really just up to the user to convert back to degrees when necessary. 

I'd question whether or not we should be including these at all, really...

Reply:
We have some constants in degrees (`alpha`, `beta`, `gamma`) `lattice_parameters` and we have to take `cos`, `sin` of these parameters- that's pretty much the only time I need them, but regardless I think those functions can go away and we can convert things on the spot when needed during calculations (using `numpy`)

Reply:
That would be my preference, but maybe I'm being too opinionated here

Reply:
cosd, sind, tand, atand, acosd ... etc, those came from Matlab and are also 'beloved' convenience features in the old hutch python. They are used daily by a lot of people and we should definitely keep them.

Reply:
btw, I'm opinionated too ... but I'll make sure to 5-star this one when I get to do my user satisfaction survey :)

Reply:
Added these back in e38daa0.